### PR TITLE
Feature/us324 restrict summary access

### PR DESCRIPTION
**Pull request by RichardProsser**

### CODE REVIEW

File: service/server.py [hunk 1]
```diff
@@ -389,6 +385,13 @@ def _username_from_header(request):
...
 def _validates_user_group(request):
     # Get user group from WebSeal headers
-    user_group = request.headers.get("iv-groups", "")
-    if "DRV" not in user_group.upper():
-        abort(404)
+    # user_group = request.headers.get("iv-groups", "")
+    # if "DRV" not in user_group.upper():
+    #     abort(404)
+    pass
+
```
Comment (Pheonnexx) [inline service/server.py:398]: This needs the uncommented code put pack in :3 :banana:

**Comment by RichardProsser** [comment]
lr-run-unit-tests: all pass

Checking type annotations with mypy
tests/test_app.py: In member "test_title_search_redirects" of class "TestTitleSearch":
tests/test_app.py, line 364: Unexpected keyword argument "search_term" for "dict"

This appears to be spurious.

**Comment by Pheonnexx** [comment]
@RichardProsser we actually ignore most of the unit tests with pep8 - same with some of the rubocop stuff for ruby.  Could you possibly rebase this branch if we are going to merge this - when eventually get this story into a sprint.

File: config.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 more_proprietor_details = os.environ['MORE_PROPRIETOR_DETAILS']
 show_full_title_data = os.environ['SHOW_FULL_TITLE_DATA'].lower() == 'true'
 show_full_title_pdf = os.environ['SHOW_FULL_TITLE_PDF'].lower() == 'true'
-title_register_summary_price = "&pound;1.20 (incl. VAT)"
+title_register_summary_price = "2.0"
```
Comment (mattgirdler) [inline config.py]: Should be 3.0 now

**Comment by mattgirdler** [comment]
Other than the summary price - LGTM

**Comment by RichardProsser** [comment]
Changed title_register_summary_price to "3.0", after rebase; units tests OK.

**Pull request merged**